diagnostic.md

```diff
@@ -5,74 +5,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+We like to have a backend if we want to be able to save things/hold data since the browser doesn't really hold that kind of info for us... Basically once an application gets to the client, it's completely separated from everything else unless we have a backend.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+It's just stuff/data that gets sent back to the client... it's more for the user than the developer. We replace views with serializers in Rails.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create
+Read
+Update
+Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+The controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+create
+index
+show
+update
+destroy
 ```
```

Comment:
Looking for this:
`Index (READ), Create (CREATE), Show (READ), Update (UPDATE), Destroy (DESTROY).`